RevenueCatUI/Plugins/Android/RevenueCatUI.androidlib/src/main/java/com/revenuecat/purchasesunity/ui/PaywallTrampolineActivity.java

```diff
@@ -109,6 +114,36 @@ private void launchPaywall(PaywallUnityOptions options) {
         }
     }
 
+    @Nullable
+    private PresentedOfferingContext mapPresentedOfferingContext(@Nullable String jsonString, @Nullable String fallbackOfferingId) {
```

Comment:
In this context `jsonString` is `offeringIdentifier` right?

Reply:
Hmm but this is a json string which gets decoded as a json object so order doesn't really matter I think?